plugin/trino-hive/src/main/java/io/trino/plugin/hive/procedure/UnregisterPartitionProcedure.java

```diff
@@ -87,6 +88,11 @@ public void unregisterPartition(ConnectorSession session, ConnectorAccessControl
 
     private void doUnregisterPartition(ConnectorSession session, ConnectorAccessControl accessControl, String schemaName, String tableName, List<String> partitionColumn, List<String> partitionValues)
     {
+        checkProcedureArgument(schemaName != null, "schema_name cannot be null");
+        checkProcedureArgument(tableName != null, "table_name cannot be null");
+        checkProcedureArgument(partitionColumn != null, "partition_columns cannot be null");
```

Comment:
Could you rename `partitionColumn` to `partitionColumns` in a separated commit or PR?

Reply:
No change is requested. Ideally, the above change should be commited first to reduce diff.